src/library/dao/trackdao.cpp

```diff
@@ -1819,7 +1825,7 @@ bool TrackDAO::isTrackFormatSupported(TrackInfoObject* pTrack) const {
     return false;
 }
 
-void TrackDAO::verifyRemainingTracks() {
+bool TrackDAO::verifyRemainingTracks(volatile const bool* pCancel) {
```

Comment:
This is the only function I see that might cause a slow down because we are doing actual IO. Here I'm ok issuing a cancel. With the others I don't see the need